### PR TITLE
documented error_message() method of Validation

### PR DESCRIPTION
**Pull request by hackoh**
document of https://github.com/fuel/core/pull/1536

### CODE REVIEW

File: classes/validation/methods.html [hunk 1]
```diff
@@ -554,6 +554,68 @@ <h4 class="method" id="method_callables">callables()</h4>
 				</table>
 			</article>
 
+			<article>
+				<h4 class="method" id="method_error_message">error_message($field, $default)</h4>
+				<p>
+					The <strong>error_message</strong> method returns specific error message or all error messages thrown during validation.
```
Comment (emlynwest) [inline classes/validation/methods.html]: See my comment on #586

**Comment by hackoh** [comment]
Please check fixes, too.

**Comment by frankdejonge** [comment]
@hackoh there is a merge conflict, can you resolve this? The method itself is merged.

**Comment by hackoh** [comment]
ok, i see.
I will resolve this tomorrow.
I'm on the way to drinking street.

**Pull request closed without merging**